payment-in-stablecoins/README.md

```diff
@@ -0,0 +1,95 @@
+# Proposal Payment in Stablecoins
+
+## Simple Summary
+
+Allows the DAO to execute proposals that send funds denominated in USD. This change addresses the ETH/USD price volatility risk both sides suffer from today, as well as the DAO's ability to perform ETH-to-USD swaps trustlessly.
+
+## Abstract
+
+This specification introduces two new contracts that facilitate stablecoin payments:
+
+- `NOU` a new ERC20 token the DAO can mint to say "I owe you this amount of USD"
+- `NounsStablecoinPayments` a contract that:
+  - allows the DAO to mint NOU tokens as payment to proposal recipients
+  - allows anyone to sell it stablecoins in exchange for ETH with some additional incentive
+  - allows proposal recipients to redeem NOU for stablecoins
+
+## Technical Specification
+
+### Overview
+
+#### `NOU` contract
+
+A new ERC20 token contract:
+
+- Inherits from OpenZeppelin's `AccessControlEnumerable`
```

Comment:
why do we need access control? and enumerable?

maybe enough to just make it Ownable and have the mint/burn functions available only to the owner?

Reply:
great point to discuss further; the reason I leaned towards this is "future proofing", since tokens tend not to change, wanted to allow the token to support use cases we might not be thinking of right now. agree it adds complexity and some risk.

Reply:
@solimander any thoughts on this?

Reply:
Not sure about enumerable, but I see how multi-role access control could be useful (e.g. transitioning to a new `NounsStablecoinPayments` contract).